enos/enos-scenario-autopilot.hcl

```diff
@@ -46,7 +46,7 @@ scenario "autopilot" {
       arm64 = "t4g.small"
     }
 
-    enable_undo_logs = matrix.undo_logs_status == "1" && semverconstraint(var.vault_product_version, ">=1.13.0-0") ? true : false
+    enable_undo_logs = matrix.undo_logs_status == "1" && try(semverconstraint(var.vault_product_version, ">=1.13.0-0"), false)
```

Comment:
I feel like Jaymala did something similar in an older PR and it had weird effects. Have you tried this to ensure stuff behaves as we expect?

Reply:
I remember adding `try` having unexpected effects, couldn't dig out the exact change though.  But if your local test breaks may be can try setting `vault_product_version` as a local variable and use that.

Reply:
So I tested the autopilot scenario on vault-enterprise, you can see it here: https://github.com/hashicorp/vault-enterprise/actions/runs/3968445250/jobs/6802027552. It seems to work fine, so I think these changes are okay.

Reply:
This condition was recently removed from the autopilot scenario, so we don't need to update this file here anymore.

Reply:
yeah it was indeed removed.